pkg/epp/metrics/metrics.go

```diff
@@ -151,6 +151,16 @@ var (
 		},
 		[]string{"name"},
 	)
+
+	inferencePoolNumReadyPods = compbasemetrics.NewGaugeVec(
+		&compbasemetrics.GaugeOpts{
+			Subsystem:      InferencePoolComponent,
+			Name:           "num_ready_pods",
```

Comment:
what is the convention here? we use running_requests, should we name this just ready_pods?

Reply:
Done.